README.md

```diff
@@ -46,4 +46,12 @@ This task watch the files localized in _src/main/resources/scss/_, if any of tho
 
 To run this task open a new console and do: 
 
-``node_modules/grunt-cli/bin/grunt watch``
\ No newline at end of file
+``node_modules/grunt-cli/bin/grunt watch``
+
+### To setup karma tests
+
+``node_modules/.bin/bower install``
```

Comment:
Could you please remove this line?, because this command is executed when you run ```npm install```. You can see in packages.json on the script attribute the execution of this command.